src/stylesheets/index.css

```diff
@@ -177,3 +264,11 @@ h1 {
   margin: 0;
   color: var(--j2-color-primary);
 }
+
+h5 {
```

Comment:
Maybe this can be its own ticket, there's probably some documentation we can do here around documenting typography methods in storybook outside of the concept of the `Components` category in storybook

Reply:
Yeah I think that's a good idea